### PR TITLE
[scala] new ensime config command

### PR DESCRIPTION
**Pull request by d1egoaz**
Thank you for contributing to Spacemacs!

Before you submit this pull request, please ensure it is against the develop branch and not master.

This message should be replaced with a description of your change.

Thank you <3
Old command doesn't exist anymore

### CODE REVIEW

**Comment by StreakyCobra** [comment]
Note: Not sure it requires hotfixing, but probably.

File: layers/+lang/scala/packages.el [hunk 1]
```diff
@@ -66,7 +66,7 @@
         "Regenerate `.ensime' file and restart the ensime server."
         (interactive)
         (progn
-          (sbt-command "gen-ensime")
+          (sbt-command ";ensimeConfig;ensimeConfigProject")
```
Comment (syl20bnr) [inline layers/+lang/scala/packages.el]: is the leading `;` required or is it a typo ?

Comment (d1egoaz) [inline layers/+lang/scala/packages.el]: It's required, see http://www.scala-sbt.org/0.13/docs/Howto-Running-Commands.html#Provide+multiple+commands+to+run+consecutively

Comment (d1egoaz) [inline layers/+lang/scala/packages.el]: With leading `;`:
```scala
sbtconsole> ;ensimeConfig;ensimeConfigProject
[info] ENSIME update.
```

Without:
```scala
sbtconsole> ensimeConfig;ensimeConfigProject
[error] Expected ID character
[error] ensimeConfig usage:
[error]
```

**Comment by syl20bnr** [comment]
Thank you 👍 
Keeping it open until it is backported to 0.105.

**Comment by syl20bnr** [comment]
Backported to 0.105.21

**Pull request closed without merging**